Mail/Views/New Message/ComposeMessageMacosToolbarView.swift

```diff
@@ -0,0 +1,141 @@
+        .italic,
+        .strikeThrough,
+        .cancelFormat
+    ]
+
+    let textItems: [EditorToolbarAction] = [
+        .unorderedList
+    ]
+
+    var allItems: [[EditorToolbarAction]] {
+        [extras, textFormats, textItems]
+    }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            HStack(alignment: .center) {
+                ForEach(allItems, id: \.self) { items in
+                    ForEach(items, id: \.self) { item in
+                        Button(
+                            action: {
+                                item.action(
+                                    textAttributes: textAttributes,
+                                    isShowingLinkAlert: &isShowingLinkAlert,
+                                    isShowingFileSelection: &isShowingFileSelection
+                                )
+                            }, label: {
+                                item.icon.swiftUIImage
+                                    .padding(IKPadding.medium)
+                            }
+                        )
+                        .buttonStyle(MacosToolbarButtonStyle(isActive: item.isSelected(textAttributes: textAttributes)))
+//                        .popoverToolbarHelp(title: item.accessibilityLabel)
+                    }
+
+                    if allItems.last != items {
+                        Divider()
+                            .padding(.vertical, value: .medium)
+                    }
+                }
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(.horizontal, IKPadding.composeViewHeaderHorizontal)
+
+            IKDivider()
+        }
+    }
+}
+
+struct MacosToolbarButtonStyle: ButtonStyle {
+    @State private var isHovered = false
+    let isActive: Bool
+
+    func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .background(isHovered ? MailResourcesAsset.hoverMenuBackground.swiftUIColor : .clear)
+            .background(isActive ? MailResourcesAsset.hoverMenuBackground.swiftUIColor : .clear)
+            .foregroundColor(isActive ? .primary : .secondary)
+            .cornerRadius(8)
+            .scaleEffect(configuration.isPressed ? 0.95 : 1.0)
+            .onHover { hovering in
+                withAnimation(.easeInOut(duration: 0.25)) {
+                    isHovered = hovering
+                }
+            }
+    }
+}
+
+struct PopoverToolbarHelp: ViewModifier {
+    @State private var isShowing = false
+    let title: String
+
+    func body(content: Content) -> some View {
+        content
+            .onHover { hover in
+                isShowing = hover
+            }
+            .popover(isPresented: $isShowing, arrowEdge: .bottom) {
+                ZStack {
+                    MailResourcesAsset.onTagExternalColor.swiftUIColor
+                        .scaleEffect(1.5)
+
+                    Text(title)
+                        .foregroundColor(.white)
```

Comment:
We use MailTextStyle to give a color and font to a text.
Here, it is a body regular with #F5F5F